src/browser/main.coffee

```diff
@@ -69,7 +69,9 @@ setupCrashReporter = ->
 setupAtomHome = ->
   return if process.env.ATOM_HOME
 
-  atomHome = path.join(app.getHomeDir(), '.atom')
+  atomHome = path.join(path.dirname(process.execPath), 'userdata')
```

Comment:
I understand... Actually I haven't installed the Atom, I just extracted the setup/nupkg to a directory and worked on this solution for portability, it doesn't have the "Update.exe", without this Atom will not auto-update, right?

But let's think that users would copy `C:\Users\kevin\AppData\Local\atom` folder with the updater. We can change the path to `'..\userdata'`, so it will stay with the updater and will not accidentally erased?